docs/services/aws-services-guide.md

```diff
@@ -430,74 +118,119 @@ kubectl apply -f stack-aws.yaml
 3. Obtain AWS credentials. (See the [Cloud Provider Credentials][cloud-creds]
    docs for more information.)
 
-#### Infrastructure Namespaces
+### Create AWS Provider
 
-Kubernetes namespaces allow for separation of environments within your cluster.
-You may choose to use namespaces to group resources by team, application, or any
-other logical distinction. For this guide, we will create a namespace called
-`app-project1-dev`, which we will use to group our AWS infrastructure
-components.
+It is essential to make sure that the AWS user credentials are configured in
+Crossplane as a provider. Please follow the steps [provider
+guide][aws-provider-guide] for more information.
 
-* Define a `Namespace` in `aws-infra-dev-namespace.yaml` and create it:
+## Create Configuration
 
-  ```yaml
-  cat > aws-infra-dev-namespace.yaml <<EOF
-  ---
-  apiVersion: v1
-  kind: Namespace
-  metadata:
-    name: aws-infra-dev
-  EOF
+To keep your resource configuration organized, start by creating a new
+directory:
 
-  kubectl apply -f aws-infra-dev-namespace.yam
-  ```
+```bash
+mkdir wordpress && cd $_
+```
 
-* You should see the following output:
+### RDS
 
-> namespace/aws-infra-dev created
+#### Network Resources
 
-#### AWS Provider
+Before you setup an RDS instance, you will need to create a subnet group for it
+to be provisioned into, as well as a security group to determine how it can be
+accessed
 
-It is essential to make sure that the AWS user credentials are configured in
-Crossplane as a provider. Please follow the steps [provider
-guide][aws-provider-guide] for more information.
+* Define an AWS `DBSubnetGroup` in `aws-dbsubnet.yaml` and create it:
+
+```bash
+cat > aws-dbsubnet.yaml <<EOF
+apiVersion: storage.aws.crossplane.io/v1alpha2 # change me!
+kind: DBSubnetGroup
+metadata:
+  name: sample-dbsubnetgroup
+spec:
+  groupName: sample_dbsubnetgroup
+  description: EKS vpc to rds
+  subnetIds:
+    - ${SUBNET_ONE_ID}
+    - ${SUBNET_TWO_ID}
+    - ${SUBNET_THREE_ID}
```

Comment:
I know this was the existing convention, but personally I feel like the bashisms intended to make it possible to copy and run this guide make it seem more intimidating than it needs to be, and could be mistaken by a casual observer as indicating that you _have to_ use bash scripting to build Crossplane YAML. Given we can now use cross-resource-references my preference would be to drop all the `cat >` and `${VARIABLES}` and just repeat things like example subnet names throughout our examples.

Reply:
hmm I definitely see your point here. However, I do think it is much faster going through the guide if you can just copy paste these commands. Long-term I think it would be cool to have text-boxes at the top where you could enter values and then all of the config is adjusted with your values. In the meantime, I would prefer to stick with this `cat` convention, but am more than happy to update if we decide to move away from it.

Reply:
I'm really not a fan, but I know this was a decision that was consciously made going into 0.3 and it doesn't _have_ to be fixed now. I'd like us to reassess whether this really makes our docs friendlier/easier though in future.

Reply:
If it is something we are definitely going to change, I would prefer doing so now rather than putting off. One nice thing about it is that you end up with all of your config in a single directory that you can `kubectl delete -f dir/` at the end if you want. Obviously you could do this anyway by just instructing the user to create files in directory, but it feels a little more definite this way.